### PR TITLE
WIP: New Colors 3: Mobile nav

### PR DESCRIPTION
**Pull request by chrisnojima**
This updates the nav on mobile
Also puts back the border on desktop so the badge has an outline

### CODE REVIEW

File: shared/app/tab-bar/index.render.desktop.js [hunk 1]
```diff
@@ -41,6 +41,7 @@ const TabBar = ({onTabClick, selectedTab, username, badgeNumbers}: Props) => (
       <TabBarButton
         className="keybase-nav"
         badgeNumber={badgeNumbers[tab]}
+        isNav={true}
```
Comment (chrisnojima) [inline shared/app/tab-bar/index.render.desktop.js]: this stuff is temporary. nav should just be its own thing and not use this component. this is so the folders tab isn't messed up

**Comment by keybase-ci-visdiff** [comment]
:no_entry: Error rendering the commits c7a36d762076...81725f6a6fbe on linux. <details><summary>:mag_right: 6 errored, 2 changed</summary>

 * Error in c7a36d762076: **Signup_Passphrase-Start-ERROR**  
   Signup_Passphrase-Start-ERROR rendered
 * Error in 81725f6a6fbe: **Signup_Passphrase-Start-ERROR**  
   Signup_Passphrase-Start-ERROR rendered
 * Error in c7a36d762076: **Signup_Passphrase-Error-ERROR**  
   Signup_Passphrase-Error-ERROR rendered
 * Error in 81725f6a6fbe: **Signup_Passphrase-Error-ERROR**  
   Signup_Passphrase-Error-ERROR rendered
 * Error in c7a36d762076: **Signup_Error-Start-ERROR**  
   Signup_Error-Start-ERROR rendered
 * Error in 81725f6a6fbe: **Signup_Error-Start-ERROR**  
   Signup_Error-Start-ERROR rendered
 * Changed: **App_Tab_Bar-Search_Active** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995327305-c7a36d762076-81725f6a6fbe-linux/App_Tab_Bar-Search_Active-c7a36d762076.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995327305-c7a36d762076-81725f6a6fbe-linux/App_Tab_Bar-Search_Active-81725f6a6fbe.png)  
   App_Tab_Bar-Search_Active comparison
 * Changed: **App_Tab_Bar-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995327305-c7a36d762076-81725f6a6fbe-linux/App_Tab_Bar-Normal-c7a36d762076.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995327305-c7a36d762076-81725f6a6fbe-linux/App_Tab_Bar-Normal-81725f6a6fbe.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995327305-c7a36d762076-81725f6a6fbe-linux/App_Tab_Bar-Normal.png)
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 0c81650958b3...7aa036666a85 introduce visual changes on linux. <details><summary>:mag_right: 3 changed</summary>

 * Changed: **Tracker-New_user_Scroll_1** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497998451168-0c81650958b3-7aa036666a85-linux/Tracker-New_user_Scroll_1-0c81650958b3.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497998451168-0c81650958b3-7aa036666a85-linux/Tracker-New_user_Scroll_1-7aa036666a85.png)  
   Tracker-New_user_Scroll_1 comparison
 * Changed: **App_Tab_Bar-Search_Active** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497998451168-0c81650958b3-7aa036666a85-linux/App_Tab_Bar-Search_Active-0c81650958b3.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497998451168-0c81650958b3-7aa036666a85-linux/App_Tab_Bar-Search_Active-7aa036666a85.png)  
   App_Tab_Bar-Search_Active comparison
 * Changed: **App_Tab_Bar-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497998451168-0c81650958b3-7aa036666a85-linux/App_Tab_Bar-Normal-0c81650958b3.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497998451168-0c81650958b3-7aa036666a85-linux/App_Tab_Bar-Normal-7aa036666a85.png)  
   App_Tab_Bar-Normal comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 0c81650958b3...1b3d11d90d1a introduce visual changes on linux. <details><summary>:mag_right: 2 changed</summary>

 * Changed: **App_Tab_Bar-Search_Active** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497999161808-0c81650958b3-1b3d11d90d1a-linux/App_Tab_Bar-Search_Active-0c81650958b3.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497999161808-0c81650958b3-1b3d11d90d1a-linux/App_Tab_Bar-Search_Active-1b3d11d90d1a.png)  
   App_Tab_Bar-Search_Active comparison
 * Changed: **App_Tab_Bar-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497999161808-0c81650958b3-1b3d11d90d1a-linux/App_Tab_Bar-Normal-0c81650958b3.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497999161808-0c81650958b3-1b3d11d90d1a-linux/App_Tab_Bar-Normal-1b3d11d90d1a.png)  
   App_Tab_Bar-Normal comparison
</summary>

File: shared/common-adapters/native-wrappers.native.js [hunk 2]
```diff
@@ -23,6 +23,7 @@ import {
   TouchableNativeFeedback as NativeTouchableNativeFeedback,
   TouchableWithoutFeedback as NativeTouchableWithoutFeedback,
   TouchableHighlight as NativeTouchableHighlight,
+  TouchableOpacity as NativeTouchableOpacity,
```
Comment (chromakode) [inline shared/common-adapters/native-wrappers.native.js]: Where is this used?

**Comment by chrisnojima** [comment]
It's not. I had it briefly when I was trying to get the touch stuff in the
nav working but I gave up. I left it. It seemed harmless.

**Pull request closed without merging**